jsapp/scss/stylesheets/partials/form_builder/_card.scss

```diff
@@ -68,11 +68,6 @@
             color: $black;
         }
 
-        // we use placeholder as error message, as empty title is not welcomed
-         &::placeholder {
-            color: $cool-red;
-            background-color: rgba($cool-red, 0.075);
-        }
```

Comment:
Please bring this back - these styles are used to display placeholder error message (you can search for `"Question label is required"` to see where it is being used) when question label is empty